weave/panels/panel_table.py

```diff
@@ -385,6 +385,8 @@ def active_data(self: Table) -> typing.Optional[dict]:
     refine_output_type=rows_single_refine,
 )
 def active_row(self: Table):
-    rows_node = _get_rows_node(self)
+    # _get_active_node uses an "activeRowForGrouping" which is relative to the unsorted table.
+    # for the index to be correct, we need to use the unsorted table.
+    rows_node = _get_rows_node(self, apply_sort=False)
```

Comment:
do we know why `activeRowForGrouping`'s index is relative to unsorted? Do we have the same problem in the `pinned_rows` code path?

Reply:
because the activeRowForGrouping gives the zero-based value of this pseudo-column, which is only equal to the index value when the table is unsorted

<img width="392" alt="Screen Shot 2023-08-21 at 3 30 56 PM" src="https://github.com/wandb/weave/assets/2769632/d54a7c87-b2a2-49be-81eb-637908ccb938">

Reply:
I think we will have the issue for pinned rows, let me check...

Reply:
yep, same problem, same fix

Reply:
verified fix for `pinned_rows()`